README.md

```diff
@@ -34,22 +34,34 @@ The hooks (`useResponsive` and `useIsMobile`) are the preferred method of using
 When possible, use the `withIsMobile` and `useIsMobile` for mobile devices detection. In the future we might use it to automatically splitting of mobile-only code.
 
 ## ResponsiveProvider Props
-| Prop               | Type    | Description                                                                   |
-| ------------------ | ------- | ----------------------------------------------------------------------------- |
-| initialMediaType   | string  | Used to mock a breakpoint in the initial state calculation. Defaults to 'xs'. |
-| defaultOrientation | string  | Used to mock orientation in initial state calculation. Defaults to null.      |
-
-## Object returned by the consumers:
-
-| Key                    | Type    | Description                                                                                                                      |
-|------------------------|---------|----------------------------------------------------------------------------------------------------------------------------------|
-| isCalculated           | Boolean | False on first render. Once true, it means all breakpoints values are based on the window.                                       |
-| mediaType              | String  | The current media type (breakpoint) name (ex: `'md'`).                                                                           |
-| lessThan               | Object  | Object containing if current window width is less than a breakpoint (usage: `lessThan.lg`).                                      |
-| greaterThan            | Object  | Object containing if current window width is greater than a breakpoint (usage: `greaterThan.lg`).                                |
-| is                     | Object  | Object containing if current window width is the at a breakpoint (usage: `is.lg`).                                               |
-| orientation            | String  | Current browser orientation (portrait, landscape or defined defaultOrientation at ResponsiveProvider, when others not available) |
-
+| Prop               | Type    | Required | Default | Description                                                                   |
+| ------------------ | ------- | ---------|---------|-------------------------------------------------------------------------------|
+| initialMediaType   | '_initial' <br>&#124;&nbsp; 'xs' <br>&#124;&nbsp; 'sm' <br>&#124;&nbsp; 'md' <br>&#124;&nbsp; 'lg'  <br>&#124;&nbsp;  'xl'| no    | 'xs'       |  Intial media type before the calculation of the real measures  |
+| defaultOrientation | 'landscape' <br>&#124;&nbsp; 'portrait'  | no    | null       | Intial oritation before the calculation of the real measures    |
+| children           | ReactNode  | yes     | -       | React component |
+| breakpoints        | { xs: string, sm: string, md: string, lg: string, xl: string }  | no    | -       | Min breakpoints     |
```

Comment:
Shouldn't the breakpoints and breakpointsMax be required? I tried to run a `ResponsiveProvider` with only `children` and got this results

```
{
mediaType: "_initial",
orientation: "landscape",
isCalculated: true,
is: {_initial: true},
greaterThan: {_initial: false},
lessThan: {_initial: false},
}
```

Does this make sense? @SoaresMG @sofiacteixeira